### PR TITLE
Update Miovision API pull script to adapt to recent changes

### PR DESCRIPTION
**Pull request by cczhu**
## What this pull request accomplishes:

- Changes the time interval used in sequential API calls in `intersection_tmc.py` to 6 hours to avoid late data gaps.
- Have `intersection_tmc.py` exit with code 1 if any exceptions is encountered when pulling raw data from the API with `pull_data`.
- Assign vehicle class `9` to `MotorizedVehicle`. Have `get_classification` raise an exception if it encounters a class that hasn't been defined in the function.
- Remove a number of trailing whitespaces.
- Adds development notebooks that confirms the use of a 6-hour time interval for API pulls, and checks that we can successfully import raw data into Postgres.

## Issue(s) this solves:

- #323
- #329
- #330

## What, in particular, needs to reviewed:

- Examination of the changes to `intersection_tmc.py` to make sure I haven't made any careless mistakes.

### CODE REVIEW

File: volumes/miovision/api/intersection_tmc.py [hunk 1]
```diff
@@ -327,23 +340,20 @@ def pull_data(conn, start_time, end_time, intersection, path, pull, key, dupes):
...
         except psycopg2.Error as exc:
             logger.exception(exc)
             sys.exit(1)
-        
+
         if pull_data:
             logger.info('Skipping aggregating and processing volume data')
         else:
-            process_data(conn, start_time, end_iteration_time)
+            process_data(conn, c_start_t, c_end_t)
```
Comment (radumas) [inline volumes/miovision/api/intersection_tmc.py:354]: Per our discussion, `process_data()` should get moved out of the `time_delta` loop and take `start_time, end_time` as parameters

**Review by radumas — CHANGES_REQUESTED** [review]
Nice changes to the README, whitespace, and great addition of notebooks.

One change wrt `process_data`

Also would be good if notebook folder was referenced in README, and I guess notebook folder should get its own README to briefly touch on what each notebook is about?

File: volumes/miovision/api/intersection_tmc.py (hunk 1 above)
Comment (cczhu) [reply volumes/miovision/api/intersection_tmc.py]: Moved code block out of the loop. Haven't made the requisite changes to `miovision_api.refresh_gapsize_lookup`, but as discussed we can save that for another PR.

**Review by radumas — APPROVED** [review]
Awesome!

**Pull request merged**